ReflectionAnalyzers/ReflectionAnalyzers.csproj

```diff
@@ -20,18 +24,25 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Version>0.1.20.0-dev</Version>
+    <Version>0.1.20-dev</Version>
+    <DevelopmentDependency>true</DevelopmentDependency>
     <Description>Analyzers for System.Reflection</Description>
-    <PackageLicenseUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</PackageLicenseUrl>
-    <RepositoryUrl>https://github.com/DotNetAnalyzers/ReflectionAnalyzers</RepositoryUrl>
-    <PackageTags>Reflection Roslyn Diagnostic Analyzer DotnetAnalyzers</PackageTags>
```

Comment:
⚠ I **do not** recommend removing this line. This is for equivalence with your current package.